### PR TITLE
remove localized system message handling

### PR DESCRIPTION
**Pull request by flav-code**

### CODE REVIEW

**Review by conorwastakenwastaken — APPROVED** [review]

File: lib/structures/Message.js [hunk 1]
```diff
@@ -244,6 +244,22 @@ class Message extends Base {
...
+            case MessageTypes.STAGE_END: {
+                data.content = `${this.author.mention} ended ${this.content}`;
+                break;
+            }
+            case MessageTypes.STAGE_SPEAKER: {
+                data.content = `${this.author.mention} is now a speaker.`;
+                break;
+            }
+            case MessageTypes.STAGE_TOPIC: {
+                data.content = `${this.author.mention} changed the Stage topic: ${this.content}`;
```
Comment (bsian03) [inline lib/structures/Message.js]:
```suggestion
                data.content = `${this.author.mention} changed the Stage topic: **${this.content}**`;
```

**Review by bsian03 — CHANGES_REQUESTED** [review]
Just this small bold topic change text. Also will consult abal about that content reassignment since we're directly replacing data from discord.
otherwise LGTM

File: lib/structures/Message.js (hunk 1 above)
Comment (flav-code) [reply lib/structures/Message.js]: image

**Comment by bsian03** [comment]
we decided the best way moving forward would be to remove all system localisation message handling, i.e. we will not be setting the message content if a particular message type required it

**Comment by flav-code** [comment]
you want me to remove all data.content from message system ?

**Comment by bsian03** [comment]
correct, the only thing that should be in content is what's reflected on discord

**Comment by MCausc78** [comment]
Personally I would make a getter like `displayedContent` or `systemContent` rather replacing data.content

**Comment by flav-code** [comment]
> Personally I would make a getter like `displayedContent` or `systemContent` rather replacing data.content

I agree

**Comment by bsian03** [comment]
neither of us saw a legitimate and proper use case for having localised system messages in the message content. I'm happy for you to discuss this with me on discord though.

**Comment by flav-code** [comment]
we could even do away with the user's call function, thus eliminating this unnecessary switching case

It would be nice to remove all functions not intended for bots from the lib, it would lighten the whole and remove unused code.

**Comment by bsian03** [comment]
I'll work on that, the PR is massively out of date so I'll have to look over everything again but it's on my list of things to do asap, unless you're happy doing it of course

**Review by bsian03 — APPROVED** [review]

**Pull request merged**